.github/workflows/style_and_docs.yaml

```diff
@@ -0,0 +1,65 @@
+name: style and docs
+
+on:
+  push:
+    branches:
+      - develop
+      - releases/**
+  pull_request:
+    branches:
+      - develop
+      - releases/**
```

Comment:
The doc/flake8 checks don't run in all the cases that they used to - from `linux_unit_tests.yaml`:

```

  push:
    branches:
      - master
      - develop
      - releases/**
  pull_request:
    branches:
      - master
      - develop
```

I think it may make sense to keep these conditions the same, so perhaps the version check could remain in a separate file?

Reply:
We removed the `master` branch recently so there is no need for that run. The current workflow adds a new trigger (run on pull requests for branches that match `releases/**`)